display/file.go

```diff
@@ -0,0 +1,134 @@
+package display
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/makeworld-the-better-one/amfora/renderer"
+	"github.com/makeworld-the-better-one/amfora/structs"
+)
+
+const maxSize = 1 * 1024 * 1024 // 1 Mb
+
+// handleFile handles urls using file:// protocol
+func handleFile(u string) (*structs.Page, bool) {
+	page := &structs.Page{}
+
+	filename := filepath.FromSlash(strings.TrimPrefix(u, "file://"))
```

Comment:
In this context, filename is used as a filepath (for os.open, read, readdir, ...).
I don't see how file URIs are relevant here, see FromSlash is run after file:// is removed from string.

I sure agree on renaming variable to path instead though

Reply:
You're right, I misunderstood. A valid file URI on Windows might be `file:///c:/WINDOWS/clock.avi`, but using `filepath.FromSlash(strings.TrimPrefix(u, "file://"))` on it will transform it into `\c:\WINDOWS\clock.avi`. I would like to be sure that Go will work properly with paths like that on Windows, seeing as it starts with a `\` and has a lowercase drive letter, but overall that function should stay in.

Reply:
> I would like to be sure that Go will work properly with paths like that on Windows

Please test this if you can and let me know the result.

Reply:
Yes it works now, did some additional checks after reading https://en.wikipedia.org/wiki/File_URI_scheme
Works fine on windows however I format it